packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -557,19 +285,16 @@ function FieldControl() {
 		view?.mediaField,
 		view?.descriptionField,
 	].filter( Boolean );
-	const visibleFieldIds = view.fields ?? [];
-	const hiddenFields = fields.filter(
+
+	// Get all regular fields (non-locked) in their original order from fields prop
+	const regularFields = fields.filter(
 		( f ) =>
-			! visibleFieldIds.includes( f.id ) &&
 			! togglableFields.includes( f.id ) &&
 			f.type !== 'media' &&
```

Comment:
How should we handle the featured image and content-preview fields? Right now with this check, we end up **not** allowing to display more than one `media` field. Should we just remove this check? 

Another thing related to the above is that before we had a specific place for `media` fields and opinionated styles. If we allow more `media` fields, should we add some basic styles to such fields to avoid layout issues (maybe super tall images etc..)..